### ISSUE TITLE
v1 - webcomponents-loader.js is not loading ShadyDOM in Safari 10.0.3

### ISSUE THREAD

**Issue by frankiefu** [webcomponents/webcomponentsjs#691]
Because of that `getRootNode` throws when running the following in Safari 10.0.3.
```
<!DOCTYPE html>
<html>
<head>
  <base href="//polygit.org/webcomponentsjs+v1/components/">
  <title></title>
  <script src="webcomponentsjs/webcomponents-loader.js"></script>
</head>
<body>

  <div></div>

  <script>
    addEventListener('WebComponentsReady', function() {
      let div = document.querySelector('div');
      div.getRootNode();
    });
  </script>

</body>
</html>
```

**Comment by azakus** [comment]
Seems to work with v1.0.0-rc.5: http://jsbin.com/fibotopuxo/edit?html,console,output

**Comment by azakus** [comment]
Whoops, I am mistaken :(

**Comment by azakus** [comment]
Closed as duplicate of #720
